src/EditorFeatures/VisualBasicTest/ConvertAnonymousTypeToClass/ConvertAnonymousTypeToClassTests.vb

```diff
@@ -570,7 +572,7 @@ End Class
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertAnonymousTypeToClass)>
         Public Async Function CapturedTypeParameters() As Task
             Dim text = "
-imports system.collections.generic
```

Comment:
The AddImportsService uses node.IsEquavalentTo, which doesn't consider casing rules for VB. Is there a better alternative to use?

Reply:
yeah, that seems like a bug in the service.  it should recognize these.  Alternatives: 

1. instead of syntax, we use semantics to see if there's already an import for teh same NS symbol.  
2. We have NameSyntaxComparer.  It is already used by the AddImport to properly sort things.  Looking at the code, it is properly case insensitive for VB.  So it seems like it's the "right" choice here.

Reply:
@CyrusNajmabadi 
I don't believe it is case insensitive for VB: It uses the TokenComparer which orders 'a' before 'A'. Note that this is an IComparer, not an IEqualityComparer, so it would be incorrect for it not to be case sensitive.

Reply:
You're correct.  Can you go the namespace-symbol route?

Reply:
I'm going to try passing in a custom TokenComparer to ImportsStatementsComparer instead.

Reply:
Does this need to fixed for this PR (and test change reverted)? Or can we file a tracking bug to fix the service?

Reply:
I believe I fixed this.

I didn't bother reverting this test as stylistically uppercase is more correct (I think?) and I've explicitly tested for lowercase elsewhere.